tests/test_basics/scripts/indexing.Rev

```diff
@@ -0,0 +1,20 @@
+## continue-on-error
+print("> x[0]")
+x[0]        
+print("> x[1]")
+x[1]
+
+print("> x[-1] = -1")
+x[-1] = -1
+print("> x[0] = 0")
+x[0] = 0        
+print("> x[1] = 1")
+x[1] = 1
+print("> x[2] = -2")
+x[2] = -2
+print(x)        
```

Comment:
Maybe add `print("> print(x)")` before this line, just for consistency's sake

Reply:
OK.